fsspec/implementations/ftp.py

```diff
@@ -126,6 +126,8 @@ def info(self, path, **kwargs):
         try:
             out = [f for f in files if f["name"] == path][0]
         except IndexError:
+            if path == "/":
+                return {"name": "/", "size": 0, "type": "dir"}
```

Comment:
Did you find this path gets inspected?

Reply:
Yes. If you run the ftp fs example in docstring with logging enabled and `ls` local mounted directory, you will find these messages in log file:
```
2021-09-26 13:00:14,778 FUSE operation getattr raised a <class 'fuse.FuseOSError'>, returning errno 2.
Traceback (most recent call last):
  File "/Users/xyb/Projects/fsspec/fsspec/implementations/ftp.py", line 127, in info
    out = [f for f in files if f["name"] == path][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/xyb/Projects/fsspec/fsspec/fuse.py", line 34, in getattr
    info = self.fs.info(path)
  File "/Users/xyb/Projects/fsspec/fsspec/implementations/ftp.py", line 131, in info
    raise FileNotFoundError(path)
FileNotFoundError: /historic/packages/RPMS/.DS_Store

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/xyb/.virtualenvs/fsspec/lib/python3.8/site-packages/fuse.py", line 734, in _wrapper
    return func(*args, **kwargs) or 0
  File "/Users/xyb/.virtualenvs/fsspec/lib/python3.8/site-packages/fuse.py", line 774, in getattr
    return self.fgetattr(path, buf, None)
  File "/Users/xyb/.virtualenvs/fsspec/lib/python3.8/site-packages/fuse.py", line 1027, in fgetattr
    attrs = self.operations('getattr', self._decode_optional_path(path), fh)
  File "/Users/xyb/.virtualenvs/fsspec/lib/python3.8/site-packages/fuse.py", line 1076, in __call__
    return getattr(self, op)(*args)
  File "/Users/xyb/Projects/fsspec/fsspec/fuse.py", line 36, in getattr
    raise FuseOSError(ENOENT)
fuse.FuseOSError: [Errno 2] No such file or directory
2021-09-26 13:00:15,150 getattr /
```
unless you add the code above.